guides/logging.txt

```diff
@@ -0,0 +1,24 @@
+TOPIC
+    logging
+
+DESCRIPTION
+    Bolt logs messages both to the console and to log files. Messages can
+    either come from Bolt's internal 'outputter' class, which logs user-facing
+    messages like progress and results, or from the 'logger' class, which logs
+    warnings, errors, and log-structured output to log files. Both of these message
+    streams are configurable.
+
+    By default, Bolt logs to the console at 'warn' level, writes a log file to
+    '<project>/bolt-debug.log' at 'debug' level, and runs with the outputter in
+    verbose mode unless you are running a plan, in which case it runs without
+    verbose (you still with me?). You can configure logging output in your
```

Comment:
```suggestion
    By default, Bolt logs to the console at 'warn' level and writes a log file to
    '<project>/bolt-debug.log' at 'debug' level. It also runs in verbose mode
    verbose mode unless you are running a plan, in which case it runs without
    verbose mode. You can configure logging output in your
```

Reply:
Aww, I like the "you still with me?". Too casual or negative?

Reply:
It's a bit risky. It's an attempt to be inclusive, but a reader could find it a bit condescending. Also, sometimes phrases like this read as an admission of guilt (ie. "We're sorry this is so complicated").

Reply:
I do like the casual-ness, but agree with Heston's latter point. To me it makes it seem like the sentence is harder to follow / the concept is more difficult than it really is.

Reply:
I think we could take the exception for plans to a new sentence (and maybe explain why):

"By default, Bolt logs to the console at 'warn' level and writes a log file to
'<project>/bolt-debug.log' at 'debug' level. Unless you are running a plan, 
Bolt runs in verbose mode by default. Bolt Plans use special formatting
depending on a plan's output, and do not use verbose mode by default."